src/SimpleWMSBody.hpp

```diff
@@ -0,0 +1,128 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2019 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef CSP_WMS_SIMPLE_BODIES_HPP
+#define CSP_WMS_SIMPLE_BODIES_HPP
+
+#include "WebMapTextureLoader.hpp"
+#include "utils.hpp"
+
+#include "../../../src/cs-core/SolarSystem.hpp"
+#include "../../../src/cs-core/TimeControl.hpp"
+#include "../../../src/cs-scene/CelestialBody.hpp"
+
+#include <VistaKernel/GraphicsManager/VistaOpenGLDraw.h>
+#include <VistaOGLExt/VistaBufferObject.h>
+#include <VistaOGLExt/VistaGLSLShader.h>
+#include <VistaOGLExt/VistaTexture.h>
+#include <VistaOGLExt/VistaVertexArrayObject.h>
+
+#include <glm/gtc/type_ptr.hpp>
```

Comment:
These are quite a lot of includes. Can some of them be transformed to forward declarations?